prepublish.sh

```diff
@@ -12,18 +12,26 @@ git pull
 
 echo "${GREEN}Cleaning directory and fresh installing..${NC}"
 rm -rf **/node_modules
+# We need to delete the package-lock.json created by npm link in design-system-scripts
+# to prevent an error later on in `lerna version`
+rm -rf **/package-lock.json
 yarn install
 
 echo "${GREEN}Building files and running tests..${NC}"
 yarn build
 yarn test
 yarn test:e2e --skipBuild
-#backstop test
 
-echo "${GREEN}Bumping version and pushing tagged commit to Github...${NC}"
-yarn lerna version --force-publish
+echo "${GREEN}Bumping version and creating tagged release commit...${NC}"
+yarn lerna version --no-push --force-publish
+
+echo "${GREEN}Pushing release commit to Github...${NC}"
```

Comment:
Originally I wanted to rely on lerna to push the release commit up to github, but since we disable pushing to master I have to manually create a release branch and push it up